multiversx_sdk_core/tokens.py

```diff
@@ -5,6 +5,7 @@
 from multiversx_sdk_core.constants import TOKEN_RANDOM_SEQUENCE_LENGTH
 from multiversx_sdk_core.errors import (BadUsageError,
                                         InvalidTokenIdentifierError)
+from multiversx_sdk_core.interfaces import IToken, ITokenIdentifierParts
 
 
 class Token:
```

Comment:
use `IToken` in `TokenTransfer` constructor?

Reply:
indeed. fixed